PyViCare/PyViCareDevice.py

```diff
@@ -15,3 +15,19 @@ def __init__(self, service: ViCareService) -> None:
     @handleNotSupported
     def getSerial(self):
         return self.service.getProperty("device.serial")["properties"]["value"]["value"]
+
+    @handleNotSupported
+    def isHeatingDevice(self):
+        return self.service.getProperty("heating")["isEnabled"]
```

Comment:
Would it be nice to catch the `PyVicareNotSupported` error internally, rather than relying on the implementation of error handling from the Users?

Reply:
Agree, this should return false if it does not exists.